### ISSUE TITLE
OpenFOAM in homebrew and linuxbrew

### ISSUE THREAD

**Issue by luzpaz** [luzpaz/FreeCAD-dependencies#17]
https://github.com/Homebrew/homebrew-core/issues/34327

**Comment by luzpaz** [comment]
@ipatch any interest in getting this to happen?

**Comment by ipatch** [comment]
yes. i'll attempt to invest some time 🤞 in this this week (monday through friday). not sure, if i'll get it fully working, but i can post my findings / work in comments here if that helps.

**Comment by ipatch** [comment]
looks like somebody has written a homebrew formula for openfoam,

https://github.com/wangsen992/homebrew-comp/blob/85292cd1f43224025ee334140525a2c43c1e7867/Formula/openfoam.rb

does freecad already support openfoam?
> apparently it does, https://forum.freecadweb.org/viewtopic.php?f=37&t=63782&p=547611#p547578

does one of the later the conda releases of freecad ship with the openfoam modules included?
> i downloaded the latest conda release (this afternoon) and did not see any openfoam related binaries packaged with the release. however, doesn't mean i didn't look in the right place.

all that aside, for macos users it appears the most active github repo related to openfoam i found is listed below

https://github.com/mrklein/openfoam-os-x

apparently there are several different versions of openfoam and licenses concerning the different versions. that repo listed above seems to do a good job for documenting most info related to openfoam.

also i came across a freecad forum thread where a particular user has setup openfoam for their macos / freecad installation.

https://forum.freecadweb.org/viewtopic.php?f=37&t=63782&p=547611#p547578

---

there seems to be some effort in getting openfoam bundled up into a homebrew tap below,

https://github.com/apjanke/homebrew-openfoam

---

at this point, not sure what the next step would be. not entirely sure if the homebrew-freecad tap should provide a openfoam formulae as there's glaring issues that will arise, the first being that openfoam appears to require a case sensitive volume to work with ie. build and run from what i understand. and the list will grow from there. but i hope this serves as a point of reference for those wanting to use openfoam on macos.
